src/media.js

```diff
@@ -1,25 +1,28 @@
-// This file is part of Vidyamantra - http:www.vidyamantra.com/
-/** @Copyright 2014  Vidya Mantra EduSystems Pvt. Ltd.
+/**
+ * This file provides functionalies of capturing, transmiting and playing audio/video.
+ * @Copyright 2014  Vidya Mantra EduSystems Pvt. Ltd.
  * @author  Suman Bogati <http://www.vidyamantra.com>
  * @author  Jai Gupta
- * This file provides functionality to capture , transmit and play audio and
- * video for multiple users.
- *
  */
 
-(function (window) {
-  let repMode; let buf; let vidId; let randomTime; let
-    cthis;
-
+function MediaWrapper(window) {
```

Comment:
Codacy found an issue: ['MediaWrapper' is defined but never used.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=6361181)